pybossa/jobs.py

```diff
@@ -850,12 +852,16 @@ def export_tasks(current_user_email_addr, short_name,
             max_size = current_app.config.get('EXPORT_MAX_SIZE', float('Inf'))
 
             if len(content) > max_size and bucket_name:
-                conn = create_connection(**current_app.config.get('S3_EXPORT_CONN', {}))
+                current_app.logger.info("uploading exporting tasks to s3...")
```

Comment:
include project id here and below when logging information about step being performed.